.mailmap

```diff
@@ -495,6 +495,8 @@ Ryan Wiedemann <Ryan1729@gmail.com>
 S Pradeep Kumar <gohanpra@gmail.com>
 Sam Radhakrishnan <sk09idm@gmail.com>
 Samuel Tardieu <sam@rfc1149.net>
+Santiago Pastorino <spastorino@gmail.com>
+Santiago Pastorino <spastorino@gmail.com> Santiago Pastorino <santiago@wyeworks.com>
```

Comment:
```suggestion
Santiago Pastorino <spastorino@gmail.com> <santiago@wyeworks.com>
```

I think this works?

Reply:
I have no idea :), I think so but the .mailmap file has tons of entries with the name repeated. Anyway going to change it.